### DISCUSSION TITLE
transfer saved places from maps.me to organic maps app

### DISCUSSION THREAD

**Discussion by lovelyzyx** [organicmaps/organicmaps#5459] [Q&A] [answered]
is it possible to transfer my saved places and folders in Maps.me app to Organic Maps app? They are now limiting msps and charging exorbitant fees. I use an iphone and need dozens if saved places transferred to the new app. thanks

**Comment by biodranik** [comment, marked as answer]
You have several options:
1. Export each bookmark list from maps.me and immediately open it in Organic Maps.
2. Copy the bookmarks folder with kmb files from maps.me manually using Finder and [iOS File Sharing](https://support.apple.com/en-us/HT210598) to your computer, and then convert files to kml and import them. I can help with conversion, or you can build kmb_to_kml tool from organicmaps repository and do it yourself.

**Comment by lovelyzyx** [comment]
Many thanks.

However, there is no way, as far as I’m aware, of obtaining the bookmarks in maps.me <http://maps.me/>. They are inside the app on the iphone, but there is no visible option to access them or export them

Freddy Silva

**Reply by biodranik** [reply]
There are two ways as I described above. Let's focus on the first one. Open maps.me. Open bookmarks dialog. Open the bookmarks list that you want to export. Select ... at the top right. Then select Share. Now you can select Organic Maps in the list of apps to import bookmarks.

**Comment by lovelyzyx** [comment]
You are a genius Sir, thank you


Freddy Silva
